src/tests/ftest/checksum/csum_error_logging.py

```diff
@@ -64,11 +64,17 @@ def test_csum_error_logging(self):
                 if 'tgt_ids' not in device or 'uuid' not in device:
                     self.fail(
                         'Missing uuid and/or tgt_ids info from dmg storage query list devices')
+                if 'role_bits' not in device:
+                    self.fail(
+                        'Missing role bits info from dmg storage query list devices')
```

Comment:
Will this work for PMEM mode?

Reply:
I checked this with PMEM mode, output is same, so it should work.
--------
wolf-150
--------
  Devices
    UUID:f8edbb4f-21c9-4ce9-b7d3-79481fefb511 [TrAddr:0000:67:00.0]
      Roles:data Targets:[0] Rank:0 State:NORMAL LED:NA